publish-extensions.js

```diff
@@ -59,7 +59,7 @@ const readFile = util.promisify(fs.readFile);
       if (extension.checkout) {
         await exec(`git checkout ${extension.checkout}`, { cwd: '/tmp/repository' });
       }
-      const location = path.join('/tmp/repository', extension.location || '.');
+      const location = path.join('/tmp/repository', extension.installDir || extension.location || '.');
```

Comment:
I don't see the point of `installDir` when there is already `location`. Please remove it.